src/go/README.md

```diff
@@ -0,0 +1,25 @@
+# Go APIView token file generator
+
+## Getting started
+
+### Prerequisites
+
+- Go 1.13 or above.
+
+### Generate the tool
+
+To create the command (this will output an executable), run in the `/go` directory:
```

Comment:
I would reword this slightly, something like.

"To build the tool, execute the following command from the `./src/go` directory."

Reply:
Done